bank.rb

```diff
@@ -152,8 +152,8 @@ def self.owner_account
       sample = CSV.read("./support/account_owners.csv")
```

Comment:
In the owner code above, I would remove the space between the method name and the open parenthesis so like:
`def self.create_owner (person_array)` would be `def self.create_owner(person_array)`